styles/tailwind.css

```diff
@@ -41,6 +41,13 @@ p {
 article ul {
   @apply list-disc;
 }
+li ul {
+  @apply list-inside;
+  list-style-type: circle;
+}
+aside ul {
+  list-style-type: none;
+}
```

Comment:
Indented lists use the following HTML format:
```
<ul>
<li></li>
<li>
   <ul>
     <li></li>
   <ul>
</li>
</ul>
```

So to style them, we select `li ul` and then apply the styles. Here I used the `list-inside` Tailwind utility classes to indent the child list and then the CSS property `list-style-type` to give the bullets that hollow circle look.

However, this affects the `docs` sidebar, which is wrapped in an `aside` element, so I set any `ul` within an `aside` element to have no `list-style-type`, which hides the bullets.